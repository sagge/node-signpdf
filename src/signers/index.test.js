import * as helpers from './index';

describe('Signers index', () => {
    it('Exports expected signers', () => {
        expect(Object.keys(helpers)).toMatchSnapshot();
    });
});
