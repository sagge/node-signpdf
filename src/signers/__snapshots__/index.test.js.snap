// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signers index Exports expected signers 1`] = `
Array [
  "P12Signer",
]
`;
